viewer/viewer.js

```diff
@@ -7686,24 +7686,124 @@ function buildHuntOptions (huntId, hunt) {
   return options;
 }
 
+// if we couldn't retrieve the seession, skip it but add it to failedSessionIds
+// so that we can go back and search for it at the end of the hunt
+function continueHuntSkipSession (hunt, huntId, session, sessionId, searchedSessions, cb) {
+  if (!hunt.failedSessionIds) {
+    hunt.failedSessionIds = [ sessionId ];
+  } else {
+    if (hunt.failedSessionIds.length > 10000) { // TODO ECR - is this a good limit? ES doesn't specifically have an array size limit, but it does get costly to update a document if there is a large field
```

Comment:
@awick what do you think the limit should be?

Reply:
10k seems a fine place to start

Reply:
👍